Cargo.toml

```diff
@@ -30,7 +32,7 @@ panic-on-logger-errors = []
 
 [dependencies]
 bitflags = "1.3.1"
-log = { version = "0.4.5", default-features = false }
+log = { version = "0.4.17", default-features = false }
```

Comment:
Setting the lower version number here is intentional, see https://github.com/rust-osdev/uefi-rs/pull/482

tl;dr Cargo is going to pick the latest patch version anyway, so for most users changing it from 0.4.5 to 0.4.17 will have no effect. For users that build in a no-network environment that e.g. has 0.4.15 but not 0.4.17, setting the lower value here makes things not break unnecessarily.

Reply:
I see, thanks!